packages/@o3r/core/schematics/ng-add/utils/linter/index.ts

```diff
@@ -0,0 +1,25 @@
+import { SchematicContext } from '@angular-devkit/schematics';
+import { askConfirmation } from '@angular/cli/src/utilities/prompt';
+
+export const shouldOtterLinterBeInstalled = async (context: SchematicContext): Promise<boolean> => {
+  const linterPackageName = 'eslint';
+  let useOtterLinter = false;
+  try {
+    require.resolve(`${linterPackageName}/package.json`);
+    if (context.interactive) {
+      useOtterLinter = await askConfirmation(`You already have eslint installed. Would you like to add otter config rules for eslint ?
+Otherwise, you can add them later via this command: ng add @o3r/eslint-config-otter`, true);
+      if (!useOtterLinter) {
+        context.logger.info(`
+          You can add otter linter config rules later to the project via this command:
+          ng add @o3r/eslint-config-otter
+        `);
+      }
+    }
+
+  } catch {
+    context.logger.debug('eslint package not installed. Skipping otter linter phase !');
```

Comment:
if there is no `eslint` we can propose them to install it with `@o3r/eslint-config-otter` no ?

Reply:
This is only a debug log. I did not want to display something to the normal user.